src/Gizmos/babylon.boundingBoxGizmo.ts

```diff
@@ -105,6 +105,30 @@ module BABYLON {
             this._PivotCached--;
         }
 
+        /**
+         * @hidden
+         * Due to float precisiion, scale of a mesh may be uniform but float values are off by a small fraction
+         * Check if uniform within 3 decimal places to account for this
+         */
+        public static _isNonUniform(vec: Vector3) {

```

Comment:
Does this make sense to do?

Reply:
perhaps move it to math? and let the user define the epsilon